src/doc/reference/glossary/determinacy_race.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Determinacy race
+tags: determincacy race
+---
+A {% defn "race condition" %} that occurs when two
+parallel strands access the same
```

Comment:
I assume we want this first instance of "strand," rather than the later instance, to link to the glossary entry.

Also, it's more precise to say "logically parallel."  Determinacy races exist regardless of how logically parallel strands end up getting scheduled and executed on processor cores.